README.md

```diff
@@ -1,10 +1,10 @@
-# `kcp` provides a true multi-tenant Kubernetes control plane for workloads on many clusters
+# `kcp` provides a multi-tenant Kubernetes API service and a transparent multi-cluster workload service
 
 ![build status badge](https://github.com/kcp-dev/kcp/actions/workflows/ci.yaml/badge.svg)
 
-`kcp` is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that is divided into multiple "[logical clusters](docs/investigations/logical-clusters.md)" that enable multitenancy of cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
+`kcp` has two layers.  The lower layer is a generic [CustomResourceDefinition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (CRD) apiserver that is divided into multiple "[logical clusters](docs/investigations/logical-clusters.md)" that enable multitenancy of cluster-scoped resources such as CRDs and Namespaces. Each of these logical clusters is fully isolated from the others, allowing different teams, workloads, and use cases to live side by side.
```

Comment:
I think instead of layers we should use a different word - maybe "components" or "components".  Transparent multi-cluster is a component that integrates with the control plane, and needs some semi-novel API access patterns and concepts from Kube.  It's the "killer app" to drive usage of the control plane.  Control plane will not be fundamentally aware of transparent-multicluster (control plane is useful without it).  If tmc struggles, we might separate it out.  Ideally everything service can be completely layered on top of control plane (either as a code plugin or a layer on top).

Reply:
I would probably use `kcp` to describe control plane.  I would describe tmc as a subsystem or sub service.

Reply:
I do not understand https://github.com/kcp-dev/kcp/pull/622#discussion_r824055662 ; it starts by disliking the word "layers" but the goes on to describe an arrangement that is exactly layering.

This "kcp" repo includes tmc, so it seems odd to me to define "kcp" to mean something narrower.

Reply:
tmc is in the repo to allow fast iterations. There is no technical reason we can't move it to kcp-dev/workloads eventually. Maybe we will.

tmc is also a tool to ensure what we build in kcp core (I like that term) is suitable for complex services. E.g. note that in v0.5 tmc uses APIExport/APIBindings, i.e. the core primitives every other service will use.